### ISSUE TITLE
Need more whitelist functionality

### ISSUE THREAD

**Issue by tdvtdv** [thejeffreystone/hassio_addons#55]
There are these two devices with the same Id and channel.
In fact, it is one device, but is seen by two different devices. 
It is necessary to mix them somehow or to be able to filter, because for RTGR328N updates come rarely, but it is seen in HA and there is no possibility to see the second device RTGN318
![image](https://github.com/thejeffreystone/hassio_addons/assets/46950849/5fb7095c-c2d1-4043-a67b-da2f30cc2444)

**Comment by thejeffreystone** [comment]
Yeah. I need to over haul the whole thing. It was kind of a Frankenstein project
